test/metabase/test/data/generic_sql.clj

```diff
@@ -300,27 +352,40 @@
       (when (seq statement)
         (default-execute-sql! driver context dbdef (s/replace statement #"⅋" ";"))))))
 
-(defn- create-db! [driver {:keys [table-definitions], :as dbdef}]
-  ;; Exec SQL for creating the DB
-  (execute-sql! driver :server dbdef (str (drop-db-if-exists-sql driver dbdef) ";\n"
-                                          (create-db-sql driver dbdef)))
-  ;; Build combined statement for creating tables + FKs
-  (let [statements (atom [])]
-    ;; Add the SQL for creating each Table
+(defn- create-db!
+  ([driver database-definition]
+    (create-db! driver database-definition false))
+  ([driver {:keys [table-definitions], :as dbdef} skip-drop-db]
+    (if-not skip-drop-db
```

Comment:
For conditionals that don't have an `else` clause you should just use `when`/`when-not` instead of `if`/`if-not`. https://github.com/bbatsov/clojure-style-guide#when-not-instead-of-single-branch-if-not